src/Action/NotFoundAction.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\Action;
+
+use Symfony\Component\HttpKernel\Exception\NotFoundHttpException;
+
+/**
+ * Not found action.
+ *
+ * @author Antoine Bluchet <soyuka@gmail.com>
```

Comment:
Unnecessary? 👻

Reply:
it was to be consistent with the other actions